doc/tests/manual/reports/230725.md

```diff
@@ -0,0 +1,292 @@
+# Manual Test Report (pre v0.3 case study release)
+
+## General
+
+- Tester: Aydan Namdar Ghazani
+- Date/Time: 25 July 2023
+- Duration: 15 minutes
+- Commit/Tag: [af6b307](https://github.com/ElektraInitiative/PermaplanT/tree/af6b3070d285f273e6c173236a3c634eb4c827c3)
+- Planned tests: **13**
+- Executed tests: **13**
+- Passed tests: **10**
+- Failed tests: **3**
+
+## [Error Analysis](../README.md#report-header)
+
+- TC-006: Might be a faulty testcase.
+
+- TC-008: The test case worked, just the image loading error appeared.
+
+- TC-009: Same as 008.
```

Comment:
I think we should introduce something in between failed and passed?

Reply:
Yes, I had the same feeling, lets do this.